tests/cache/tests.py

```diff
@@ -1117,6 +1118,30 @@ def test_delete_many_num_queries(self):
         with self.assertNumQueries(1):
             cache.delete_many(['a', 'b', 'c'])
 
+    def test_cull_num_queries(self):
+        """
+        There should be only one COUNT query
+        """
```

Comment:
Not sure this adds a lot of value, the test is already pretty self-documenting.

Reply:
OK. I like docs, but I'll pull it.